app/views/layouts/application.html.erb

```diff
@@ -3,35 +3,45 @@
   <head>
     <title>Breakfast Club</title>
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <%= stylesheet_link_tag 'application', media: 'all' %>
       <%= stylesheet_link_tag 'welcome' if controller_name == 'homes' %>
-    <%= csrf_meta_tags %>
```

Comment:
Removing your `csrf_meta_tags` is not recommended. How will your site protect itself against cross-site request forgery without them?